components/footer/footer.jsx

```diff
@@ -76,26 +77,30 @@ export default class Footer extends React.Component {
   }
   render() {
     return (
-      <footer className="site-footer dark-theme py-md-5">
+      <footer className="site-footer dark-theme py-5">
         <div className="container">
           <div className="row">
-            <div className="col-md-5">
-              <div className="my-5 my-md-0 d-md-flex align-items-md-end h-100">
-                <a href="https://foundation.mozilla.org" className="logo">
-                  <img
-                    src="../assets/svg/mozilla-block-white.svg"
-                    alt="Mozilla Foundation Home Page"
-                  />
+            <div className="col-md-5 col-lg-6">
+              <NewsletterSignUp
+                buttonPosition="side"
+                ctaHeader="Want to get smarter about your online life?"
```

Comment:
Per https://github.com/mozilla/foundation.mozilla.org/issues/3161 let's remove this and stick to the default